combineLoaders.js

```diff
@@ -2,10 +2,22 @@ var qs = require('qs');
 
 function combineLoaders(loaders) {
   return loaders.map(function(loaderEntry) {
-    var query = qs.stringify(loaderEntry.query, { arrayFormat: 'brackets', encode: false });
+    // Careful, won't work if a string is created with `new String('my-loader')`
```

Comment:
why would anyone do this? there's no practical reason to use the boxed form of primitives in JS.